src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -1028,6 +1028,7 @@ components:
       type: string
       enum:
         - surname
+        - applicationReference
```

Comment:
Could also add a parameterised test in `AedSearchByParametersAreValidConstraintValidatorTest`

Reply:
Ah, nice one. I didnt know that that test class existed! Yes, I'll update it 👍

Reply:
Done